src/webapi/document.rs

```diff
@@ -67,4 +69,54 @@ impl Document {
             ).into_reference_unchecked()
         }
     }
+
+    /// Returns the <body> or <frameset> node of the current document, or null if no such element exists.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Document/body)
+    // https://html.spec.whatwg.org/#the-document-object:dom-document-body
+    pub fn body( &self ) -> Option< HtmlElement > {
+        unsafe {
+            js!(
+                return @{self}.body;
+            ).into_reference_unchecked()
+        }
+    }
+
+    /// Returns the <head> element of the current document. If there are more than one <head>
+    /// elements, the first one is returned.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Document/head)
+    // https://html.spec.whatwg.org/#the-document-object:dom-document-head
+    pub fn head( &self ) -> Option< HtmlElement > {
+        unsafe {
+            js!(
+                return @{self}.head;
+            ).into_reference_unchecked()
+        }
+    }
+
+    /// Gets the title of the document.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Document/title)
+    // https://html.spec.whatwg.org/multipage/semantics.html#the-title-element
```

Comment:
This should link to the IDL: `https://html.spec.whatwg.org/#the-document-object:document.title`